cookbooks/bcpc-hadoop/recipes/resource_manager.rb

```diff
@@ -32,22 +32,54 @@
   not_if "hdfs dfs -test -d /var/log/hadoop-yarn", :user => "hdfs"
 end
 
+bash "create-hdfs-yarn-ats-log" do
+  code <<-EOH
+  hdfs dfs -mkdir -p /var/log/ats && hdfs dfs -chmod 1777 /var/log/ats && hdfs dfs -chown yarn:mapred /var/log/ats
+  hdfs dfs -mkdir -p /var/log/ats/active && hdfs dfs -chmod 1777 /var/log/ats/active && hdfs dfs -chown yarn:mapred /var/log/ats/active
+  hdfs dfs -mkdir -p /var/log/ats/done && hdfs dfs -chmod 0700 /var/log/ats/done && hdfs dfs -chown yarn:mapred /var/log/ats/done
+  EOH
+  user "hdfs"
+  only_if { node.roles.include?("BCPC-Hadoop-Head-YarnTimeLineServer") }
```

Comment:
I'm stylistically opposed to the role's being used for action at a distance (e.g. instead of a feature flag) but this shouldn't cause an issue so just explaining my thought.